.eslintrc.js

```diff
@@ -1,11 +1,72 @@
 module.exports = {
     extends: ['expensify', 'prettier'],
-    rules: {
-        // Allow JSX to be written in any file ignoring the extension type
-        'react/jsx-filename-extension': 'off'
-    },
-    plugins: ['jest'],
-    env: {
-        "jest/globals": true
-    }
+    parser: '@typescript-eslint/parser',
+    overrides: [
+        {
+            files: ['*.js', '*.jsx'],
+            settings: {
+                'import/resolver': {
+                    node: {
+                        extensions: ['.js', '.jsx', '.ts', '.tsx'],
+                    },
+                },
+            },
+            rules: {
+                // Allow JSX to be written in any file ignoring the extension type
+                'react/jsx-filename-extension': 'off',
+                'rulesdir/no-api-in-views': 'off',
+                'rulesdir/no-multiple-api-calls': 'off',
+                'rulesdir/prefer-import-module-contents': 'off',
+                'no-constructor-return': 'off',
+                'import/extensions': [
+                    'error',
+                    'ignorePackages',
+                    {
+                        js: 'never',
+                        jsx: 'never',
+                        ts: 'never',
+                        tsx: 'never',
+                    },
+                ],
+            },
+        },
+        {
+            files: ['*.ts', '*.tsx'],
+            extends: ['expensify', 'plugin:@typescript-eslint/recommended', 'plugin:@typescript-eslint/stylistic', 'plugin:import/typescript', 'prettier', 'plugin:prettier/recommended'],
+            plugins: ['react', 'import', '@typescript-eslint'],
```

Comment:
we should also add `plugin:you-dont-need-lodash-underscore/all`. I know it makes the diff much bigger but we need to drop lodash/underscore from ExpensiMark for react-native-live-markdown, so we might as well stick with our standard TS style here

Reply:
Sure! I'm going to add it today 😄